Description: Final changes are made to core/deserialization/build.gradle

---

--- /dev/null
+++ b/core/deserialization/build.gradle
@@ -0,0 +1,27 @@
+plugins {
+    id("org.jetbrains.kotlin.jvm")
+    id("jps-compatible")
+}
+
+jvmTarget = "1.6"
+javaHome = rootProject.ext["JDK_16"] as String
+
+dependencies {
+    compile(project(":core:metadata"))
+    compile(project(":core:util.runtime"))
+    compile(project(":core:descriptors"))
+    compile(DependenciesKt.customCommonDep(project,"javax.inject"))
+}
+
+sourceSets {
+    "main" {
+    	java.srcDirs("src")
+		resources.srcDir("resources")
+	}
+    "test" {}
+}
+
+tasks.withType(JavaCompile) {
+    sourceCompatibility = "1.6"
+    targetCompatibility = "1.6"
+}
--- a/core/deserialization/build.gradle.kts
+++ /dev/null
@@ -1,24 +0,0 @@
-plugins {
-    kotlin("jvm")
-    id("jps-compatible")
-}
-
-jvmTarget = "1.6"
-javaHome = rootProject.extra["JDK_16"] as String
-
-dependencies {
-    compile(project(":core:metadata"))
-    compile(project(":core:util.runtime"))
-    compile(project(":core:descriptors"))
-    compile(commonDep("javax.inject"))
-}
-
-sourceSets {
-    "main" { projectDefault() }
-    "test" {}
-}
-
-tasks.withType<JavaCompile> {
-    sourceCompatibility = "1.6"
-    targetCompatibility = "1.6"
-}
